staticaddr/loopin/fsm.go

```diff
@@ -355,7 +355,7 @@ func (f *FSM) Warnf(format string, args ...interface{}) {
 // Errorf logs an error message with the loop-in swap hash.
 func (f *FSM) Errorf(format string, args ...interface{}) {
 	if f.loopIn == nil {
-		log.Errorf(format, args...)
+		log.Infof(format, args...)
```

Comment:
Is there a strong reason to change to `Infof`?

Reply:
in the case of error I guess better to keep it. Will change.